### PR TITLE
Use newest language version in scripting compilers

### PR DESCRIPTION
**Pull request by drognanar**
Makes the vb scripting compiler use the newest compiler version.
This enables features such as multi line string support or interpolated strings in a REPL.

### CODE REVIEW

**Comment by drognanar** [comment]
FYI @dotnet/roslyn-interactive

File: src/Scripting/VisualBasic/VisualBasicScriptCompiler.vb [hunk 1]
```diff
@@ -13,7 +13,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Scripting
 
         Public Shared ReadOnly Instance As ScriptCompiler = New VisualBasicScriptCompiler()
 
-        Private Shared ReadOnly s_defaultOptions As VisualBasicParseOptions = New VisualBasicParseOptions(languageVersion:=LanguageVersion.VisualBasic11, kind:=SourceCodeKind.Script)
+        Private Shared ReadOnly s_defaultOptions As VisualBasicParseOptions = New VisualBasicParseOptions(kind:=SourceCodeKind.Script)
```
Comment (KevinH-MS) [inline src/Scripting/VisualBasic/VisualBasicScriptCompiler.vb]: We should fix C# too.

**Comment by amcasey** [comment]
That must be from when scripting was ahead of master.

**Comment by amcasey** [comment]
:+1:, modulo @KevinH-MS's comment.

**Comment by genlu** [comment]
:+1:

**Comment by KevinH-MS** [comment]
Late, but :+1:

**Pull request merged**